libs/ui/elements/src/lib/pagination/pagination.component.html

```diff
@@ -0,0 +1,36 @@
+<div style="width: 100%; height: 4rem; display: flex; justify-content: center">
+  <div
+    class="capitalize text-white flex justify-center items-center p-3 rounded-lg mr-3 cursor-pointer"
+    style="background-color: #8cc630"
```

Comment:
no it's `text-secondary`

Reply:
text-secondary doesn't work, should I add it to the tailwind config?

Reply:
did you test `bg-secondary` ? should work